posthog/api/event_definition.py

```diff
@@ -27,4 +34,25 @@ class EventDefinitionViewSet(
     search_fields = ["name"]
 
     def get_queryset(self):
+        if self.request.user.organization.is_feature_available("event_property_collaboration"):  # type: ignore
+            from ee.models.event_definition import EnterpriseEventDefinition
+
+            return self.filter_queryset_by_parents_lookups(EnterpriseEventDefinition.objects.all()).order_by(
+                self.ordering
+            )
         return self.filter_queryset_by_parents_lookups(EventDefinition.objects.all()).order_by(self.ordering)
+
+    def get_serializer_class(self) -> Type[serializers.ModelSerializer]:
+        serializer_class = self.serializer_class
+        if self.request.user.organization.is_feature_available("event_property_collaboration"):  # type: ignore
```

Comment:
Technically you could have the feature available but not have the `ee` app installed which would cause an unhandled exception here.

Reply:
Can you elaborate a little on this situation? And will a try except be fine here?

Reply:
Well it's an edge case, that the import from `ee` fails because you don't have the folder available (such as if you cloned from the `posthog-foss` repo). Looking at this with fresh eyes, letting the ImportException be raised normally should work fine